desktop/apps/consignments2/client/actions.js

```diff
@@ -125,6 +135,7 @@ export function completeSubmission () {
       dispatch(updateSubmission(submissionResponse.body))
       dispatch(push('/consign2/submission/thank_you'))
     } catch (err) {
+      dispatch(stopLoading())
       dispatch(updateError('Unable to submit at this time.'))
       console.error('error!', err)
```

Comment:
Yeah, I suppose this was for debugging purposes (didn't want to do nothing). I like your suggestion to make it more verbose!